pkg/resource/podtemplatespec.go

```diff
@@ -305,8 +305,8 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, proxyLister configl
 
 	resources := corev1.ResourceRequirements{
 		Requests: corev1.ResourceList{
-			corev1.ResourceCPU:    resource.MustParse("100m"),
-			corev1.ResourceMemory: resource.MustParse("256Mi"),
+			corev1.ResourceCPU:    resource.MustParse("1"),
+			corev1.ResourceMemory: resource.MustParse("1Gi"),
```

Comment:
hm, that's a big cpu number.  we've been working lately to reduce the requests most components are making down to something closer to what they are actually using normally because our footprint is getting huge and its preventing control plane workloads from getting scheduled.  

Do we have prometheus data showing the registry will normally need this much cpu?  Do we just need t o provide better tuning instructions to admins suggesting they raise these values under certain circumstances?

Reply:
@bparees what is the normal usage? An empty cluster that doesn't use the registry at all, or our CI cluster that heavily relies on the registry?

Reply:
i think we need to aim for some sort of useful default that covers typical week 1 usage for a customer.  So not the "no usage" case, but definitely not the api.ci usage.  api.ci needs to tune their cluster to their utilization.